### PR TITLE
fix: Remove unused code

### PR DESCRIPTION
**Pull request by wsdt**
fix: Remove unused code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,7 +15,6 @@ coverage
 # vim
 *.swp
 
-.env
```
Comment (wsdt) [inline .gitignore:18]: duplicate

File: cypress/e2e/helpers/bridge.ts [hunk 2]
```diff
@@ -12,36 +12,6 @@ export default class Bridge extends Page {
     this.title = 'Bridge'
   }
 
-  switchNetworkType(network: string, isTestnet: boolean, newNetwork: boolean) {
```
Comment (sk-enya) [inline cypress/e2e/helpers/bridge.ts:15]: Please do not remove the changes from cypress, it's intentional, jesus is working on it.

File: cypress/e2e/helpers/bridge.ts [hunk 3]
```diff
@@ -69,60 +39,6 @@ export default class Bridge extends Page {
       .should('be.greaterThan', 0)
   }
 
-  bridgeToken(tokenSymbol: string, amount: string, destinationLayer: Layer) {
```
Comment (sk-enya) [inline cypress/e2e/helpers/bridge.ts:72]: same here!

**Review by sk-enya — APPROVED** [review]
Looks good, please revert the cypress change then we are good to merge.

**Pull request merged**